serial_protocol.md

```diff
@@ -186,3 +192,135 @@ Used to stop listening to events at a given path. This must be the same path pro
 	>>BEGIN_STREAM /user/aturing
 	>>END_STREAM /user/aturing
 	<<+OK
+
+#Messaging Protocol
+
+## BEGIN_MESSAGING 
+Called to start communicating with Firebase Cloud Messaging, it requires a server key.
+###Usage
+	BEGIN %Server_Key%
+###Response
+	+OK - Accepted initialization parameters
+###Examples
+	>> BEGIN https://samplechat.firebaseio-demo.com
+	<< +OK
+
+## MESSAGE_TO_USERS
+Called to start composing a message to users by registration id. You can specify as many as you wish separated by spaces on the same line. 
```

Comment:
I don't have an objection on having a multiple line command if developer always need to tweak every single option between message. But if the registration id (for example) rarely change between multiple message (like say the host and credentials doesn't change between multiple SET or GET to the db) it might make sense imo to set a default with a separate command.

And that might apply to other option too.